Rakefile

```diff
@@ -538,7 +538,28 @@ end
 
 installed_pod_bin = File.join(BUNDLE_DESTROOT, 'bin/pod')
 file installed_pod_bin => rubygems_update_dir do
-  install_gem 'cocoapods', install_cocoapods_version
+  execute "bundling", ["rm", "-rf", WORKBENCH_DIR + "/ruby"]
+  # Can figure this one out later
+  bundler = "/Users/orta/.rvm/gems/ruby-2.1.3/wrappers/bundle"
```

Comment:
If I understand the goal of this PR correctly, then you might want to do something like the following to get the path:

    $ bundle show gem